charts/prometheus-pushgateway/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: "1.2.0"
 description: A Helm chart for prometheus pushgateway
 name: prometheus-pushgateway
-version: 1.4.2
+version: 1.5.0
```

Comment:
Needs a MINOR bump again. See https://github.com/prometheus-community/helm-charts/blob/main/charts/prometheus-pushgateway/Chart.yaml

Related: we should probably implement https://github.com/prometheus-community/helm-charts/issues/249 sooner than later. Anyone else want to weigh in on that?